### PR TITLE
Create saurabhmehta1601.json

### PR DESCRIPTION
**Pull request by saurabhmehta1601**

### CODE REVIEW

**Review by krishguptadev — CHANGES_REQUESTED** [review]
I think you have deleted the branch/repository, thus I can't suggest changes 😅

File: public/data/saurabhmehta1601.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+      "url": "https://github.com/saurabhmehta1601",
+      "icon": "github"
+    },
+    {
+    "name": "Follow me on Twitter",
+    "url": "https://twitter.com/saurabh__mehta",
+    "icon": "twitter"
+    },
+  ]
+}
```
Comment (Panquesito7) [inline public/data/saurabhmehta1601.json:17]:
```suggestion
{
  "name": "Saurabh Mehta",
  "bio": "JavaScript Full Stack Developer",
  "avatar":"https://github.com/saurabhmehta1601.png",
  "links": [
    {
      "name": "Follow me on GitHub",
      "url": "https://github.com/saurabhmehta1601",
      "icon": "github"
    },
    {
      "name": "Follow me on Twitter",
      "url": "https://twitter.com/saurabh__mehta",
      "icon": "twitter"
    },
  ]
}
```

**Review by shreyaansjain06 — CHANGES_REQUESTED** [review]
Please see the comments above

**Comment by saurabhmehta1601** [comment]
Thanks for your quick response . Guess,  I just realized that deleting my own fork will delete it also from pull request . I think I will be  forking this repo again and making another pull request , Now I am closing this pull request .

**Pull request closed without merging**